index.html

```diff
@@ -842,6 +848,18 @@ <h3>Endpoints</h3>
   <td><a>Find Elements From Element</a></td>
  </tr>
 
+ <tr>
+  <td>POST</td>
+  <td>/session/{<var>session id</var>}/shadow/{shadow id}/element</td>
```

Comment:
Should this be this 
```suggestion
  <td>/session/{<var>session id</var>}/shadow/{<var>element id</var>}/element</td>
```

Reply:
Should it? None of the other “Find Element(s) from X” have `<var>` markers around the secondary URL parameter. Also, since the command is “Find Element from Shadow Root”, the secondary URL parameter should be for the known shadow root ID, not the element ID. Unless we want to rewrite so that shadow roots aren’t tracked separately from their elements...

Reply:
I think `{<var>shadow id</var>}` is sufficient.

Reply:
Will add `<var>` tags around URL parameters.